cap/modules/schemas/config.json

```diff
@@ -0,0 +1,100 @@
+{
+    "name": "Configuration",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "type": "object",
+    "properties": {
```

Comment:
most of these fields should be required. please check the ones that are needed and adapt